apps/ensindexer/src/plugins/efp/efp.plugin.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * The EFP plugin describes indexing behavior for the Ethereum Follow Protocol.
+ *
+ * NOTE: this is an early version of the experimental EFP plugin and is not complete or production ready.
+ */
+
+import type { ENSIndexerConfig } from "@/config/types";
+import {
+  type ENSIndexerPlugin,
+  activateHandlers,
+  makePluginNamespace,
+  networkConfigForContract,
+  networksConfigForChain,
+} from "@/lib/plugin-helpers";
+import { DatasourceName } from "@ensnode/ens-deployments";
+import { PluginName } from "@ensnode/ensnode-sdk";
+import { createConfig } from "ponder";
+
+const pluginName = PluginName.EFP;
+
+// Define the Datasources required by the plugin
+const requiredDatasources = [DatasourceName.EFPBase];
+
+// construct a unique contract namespace for this plugin
+const namespace = makePluginNamespace(pluginName);
+
+// config object factory used to derive PluginConfig type
```

Comment:
```suggestion
// config object factory used to derive PonderConfig type
```

Please apply this update to all plugins